vuepress/docs/next/docs/ecr/ecr-bundle-details.md

```diff
@@ -99,62 +105,51 @@ Here is an example of a plugin descriptor:
+    permissions: # a list of Keycloak clientIds / roles to bind to one another
       - clientId: realm-management
         role: manage-users
       - clientId: realm-management
         role: view-users
         
 ::: tip  
- Entando uses the `healthCheckPath` to monitor the health of the plugin. A plugin in an Entando bundle can use any technology as long as it provides a health check service and configures it via the `healthCheckPath`. This path needs to be specified in the descriptor file and return an HTTP 200 or success status. This can be implemented by a Java service included with the Entando Blueprint in the Spring Boot application. You can also [use a Node.js service as shown here](https://github.com/entando-samples/ent-project-template-node-ms/blob/main/src/main/node/controller/health-controller.js). 
+ Entando uses the `healthCheckPath` to monitor the health of the plugin. A plugin in an Entando Bundle can use any technology, as long as it provides a health check service and configures it via the `healthCheckPath`. This path needs to be specified in the descriptor file and return an HTTP 200 or success status. This can be implemented by a Java service included with the Entando Blueprint in the Spring Boot application. You can also [use a Node.js service as shown here](https://github.com/entando-samples/ent-project-template-node-ms/blob/main/src/main/node/controller/health-controller.js). 
 :::
 
-### Kubernetes pod names
-
-Each plugin is deployed onto Kubernetes using composite names. The first part is created reading the descriptor file, the second one is appended autonomously by Kubernetes.
-This second part is 31 char long and each Kubernetes pod name length must be at most 63: longer name will result in the fail of the deployment.
-
-**deploymentBaseName**
-
-Initially, the first part of the pod name was generated concatenating and manipulating the `image` field value, however sometimes this approach could lead to a pod name longer than 63.
-To solve this problem, another (optional) property is available: `deploymentBaseName`.
+### Kubernetes Pod Names
 
-It accepts a string not longer than 32 and, if present, its value will be used as the first part for the pod's names, instead of the `image` one.
+Each plugin is deployed into Kubernetes using composite names. The first string is created by reading the descriptor file. The second string is appended autonomously by Kubernetes and 31 characters long. Each Kubernetes pod name length must not exceed 63 characters or the deployment will fail.
 
-In the previous example of the descriptor a possible resulting pod name will be this one in case the `deploymentBaseName` property is not present:
+**DeploymentBaseName**
 
-`entando-my-image-1-0-0-server-deployment-6f86f459wj9k`
+Initially, the first segment of the pod name was generated by concatenating and manipulating the `image` field value. However, this approach could result in a pod name longer than 63 characters.
```

Comment:
past tense because mentioning the old way of doing things (not sure it's totally necessary to give history) so changed "initially" to "previously" to make that clear